spiffworkflow-backend/src/spiffworkflow_backend/routes/process_instances_controller.py

```diff
@@ -134,6 +149,14 @@ def process_instance_run(
     return make_response(jsonify(process_instance), 200)
 
 
+def process_instance_start(
```

Comment:
Should we underscore this? That's what we've been doing when a method is called by a controller and not by the api. Helps to show what are actually api methods and what aren't.

Reply:
No problems with that.